### PR TITLE
Generate SDK automatically on main

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

File: .github/workflows/generate.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      - uses: peter-evans/create-pull-request@v3
+        with:
+          title: "chore(all): re-generate OpenAPI client(s)"
+          token: "${{ secrets.CI_USER_TOKEN }}"
+          commit-message: "chore(all): re-generate OpenAPI client(s)"
+          author: "app-services-ci <app-services-ci@users.noreply.github.com>"
+          branch: chore/generate-api-clients
+          branch-suffix: timestamp
+          reviewers: craicoverflow, wtrocki
+          delete-branch: true
```
Comment (craicoverflow) [inline .github/workflows/generate.yaml:31]:
```suggestion
          delete-branch: suffix
```

Actually that is a bug that I caused - fixed on Go SDK and forgot to check here. `true` will not work.

File: .github/workflows/generate.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  push:
+    branches:
+      - main
+    paths:
+      - ".openapi/**"
+
+jobs:
+  generate:
+    env:
+      GOPATH: /home/runner/work/app-services-sdk-go/go
```
Comment (craicoverflow) [inline .github/workflows/generate.yaml:12]: Not needed for JS SDK

File: .github/workflows/generate.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
```
Comment (craicoverflow) [inline .github/workflows/generate.yaml:17]: Not needed

File: .github/workflows/generate.yaml (hunk 1 above)
Comment (craicoverflow) [reply .github/workflows/generate.yaml:31]: Ignore me, looking at the wrong property.

**Comment by craicoverflow** [comment]
A couple of unneeded Golang related steps here. Other than its good!

**Comment by wtrocki** [comment]
CC @secondsun we need to apply the same changes to the Java SDK. 
Created issue: https://github.com/redhat-developer/app-services-sdk-java/issues/32

**Comment by wtrocki** [comment]
Let's test it

**Pull request merged**